### PR TITLE
[Doc] Add trouble shooting

### PR DESCRIPTION
**Pull request by gichan-jang**
Add a Trouble shooting to the how-to-install-taos-ci doc.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

File: ci/doc/how-to-install-taos-ci.md [hunk 1]
```diff
@@ -51,3 +51,8 @@ We will deliver event details when this hook is triggered.
 
 As a final step, press `Add webhook` button. That's all. 
 From now on, enjoy a CI world for more collaborative and productive software development!!!
+
+## Trouble shooting
```
Comment (leemgs) [inline ci/doc/how-to-install-taos-ci.md:55]: Could you replace "##" with "#"?
`s/##/#/g`

File: ci/doc/how-to-install-taos-ci.md [hunk 2]
```diff
@@ -51,3 +51,8 @@ We will deliver event details when this hook is triggered.
...
+1. I finished step 3, but the ci is not working.
+ > Please check 'github - Settings - Manage access - Collaborators'
+ > The privileged user id has to be appended by "Write" permission.
```
Comment (leemgs) [inline ci/doc/how-to-install-taos-ci.md:58]: I love this line. 👍

File: ci/doc/how-to-install-taos-ci.md (hunk 1 above)
Comment (gichan-jang) [reply ci/doc/how-to-install-taos-ci.md]: Updated.

**Review by anyj0527 — APPROVED** [review]
LGTM✔

**Comment by leemgs** [comment]
image


@gichan-jang Do you intend this result? Please check it. ❓

**Comment by gichan-jang** [comment]
> @gichan-jang Do you intend this result? Please check it. ❓

Thanks. Updated.

**Pull request merged**